.circleci/config.yml

```diff
@@ -316,6 +319,22 @@ jobs:
           command: |
             forge --version
             forge test -vvv
+  
+  monorepo_integration_test:
+    circleci_ip_ranges: true
+    docker:
+      - image: <<pipeline.parameters.default_docker_image>>
+    steps:
+      - checkout-with-mise
+      - run:
+          name: monorepo integration tests
+          environment:
+            ETH_RPC_URL: << pipeline.parameters.l1_mainnet_archive_rpc_url >>
+          command: |
+            forge build
+            forge script test/Runner.sol:Runner --sig "run(string)" lib/optimism/packages/contracts-bedrock/allocs.json --rpc-url $ETH_RPC_URL --ffi
+            export SUPERCHAIN_OPS_ALLOCS_PATH=./allocs.json
+            cd lib/optimism/packages/contracts-bedrock/ && just test-upgrade
```

Comment:
Should we batch this into a just command so it's easy to run locally when developing a task?

Reply:
Gotcha!

Reply:
https://github.com/ethereum-optimism/superchain-ops/pull/555/commits/bfd738f61b14540688e75bc6986a23674811bb0f